src/azure-cli-core/azure/cli/core/auth/identity.py

```diff
@@ -132,9 +135,15 @@ def _service_principal_store(self):
     def login_with_auth_code(self, scopes, **kwargs):
         # Emit a warning to inform that a browser is opened.
         # Only show the path part of the URL and hide the query string.
-        logger.warning("A web browser has been opened at %s. Please continue the login in the web browser. "
-                       "If no web browser is available or if the web browser fails to open, use device code flow "
-                       "with `az login --use-device-code`.", self._msal_app.authority.authorization_endpoint)
+
+        def _prompt_launching_ui(ui=None):
```

Comment:
Suggest to do it this way for it to be future-proof.
```suggestion
        def _prompt_launching_ui(ui=None, **ignored):
```

Reply:
Sure. Will use the common convention to ignore: `**_`